### DISCUSSION TITLE
Contributing to awesome-darwinia together.

### DISCUSSION THREAD

**Discussion by hackfisher** [ringdao-community-guild/discussion#14] [Ideas]
Kindly request the assistance and collaboration of our Ambassadors and dedicated community members in contributing to this repositor, @furoxr is the repo maintainer. cc @Cindy884 

https://github.com/dcdao/awesome-darwinia

**Comment by SasoLithops** [comment]
Hi!

I think it is time for community to discuss Darwinia tokenomics adjustment. I had concerns that we are not taking different variables in equasion when we did 60:40 reward split. What is happening is as predicted we get double the rewards but token price keeps on dropping. I have to admit since that decision has been made I am selling my Ring rewards weekly into DOT as I fear token price will go to many zeros. I am being honest for the love of project,not to stab it in back. I still haven't sold any Ring or Kton prior to that decision. And will switch DOT back to KTON if I belive in improvement.
A proper calculations need to be made to improve tokenomics. To calculate all the variables that could effect the potential circulating supply. I understand we can not predict the price that is NOT my intention.My intention is to improve tokenomics to make it more attractive for long term hold.
I also think from 1. June we must know who is behind Darwinia. Lots of community members have left because of no transparency who does what in Darwinia. Who is CEO, CTO, CMO.....

As for building the tech nothing to say but keep up the good work. I would also like if we could be present at more Polkadot events. I think there we could find many projects that  could potentialy use our tech. I hear these events are very rewarding from other projects...

**Reply by ken1980-web** [reply]
i just know little bit but wondering if there could burning little token weekly of monthly basis in term of community intrest and price manner...we have good use case and strong management though.

**Reply by SasoLithops** [reply]
Every comment matters our Indian community manager💪 Good point. Everything is open to discussion. Some ammount could be burned although I would rather keep tokens in community pool. If we dont give them to the market they are like burned but generate extra ammount if staked to community pool. What do you think?

**Reply by hackfisher** [reply]
This thread is for discussion about this repo maintenance, let's try to keep the discussions on topic. Off-topics should on separate threads.

For tokenomics, there is another RFC on https://github.com/darwinia-network/darwinia/discussions/1163, you can comment you ideas there.

> I also think from 1. June we must know who is behind Darwinia. Lots of community members have left because of no transparency who does what in Darwinia. Who is CEO, CTO, CMO.....

Darwinia network was founded by [Itering](https://www.itering.io/) and has consistently received unwavering support from the Itering team. Itering also developed several other famous products in the Substrate Ecosystem, including [subscan.io](https://www.subscan.io/). Itering to Darwinia is just like Parity to Polkadot. As the network develops, other groups, organization and individuals are involving and contributing to Darwinia, such as DCDAO. So in the current state， there are multiple teams are behind Darwinia, each team is independent with each other with their own management roles and governance structure, serving and supporting the network, more entities who want to join the force are surely welcome.

I also noticed that you also requested and granted the membership profile for DCDAO, I would like to express my sincerest welcome, you are also contributor behind Darwinia recognised by DCDAO now.

As Darwinia is target to be a public chain for public good purpose just like Ethereum, and have its on-chain governance mechanisms, democracy pallet, council pallet, and technical members pallet etc, I don't think we should have titles CXOs who manage the chain or decide for the network, it sound especially weird if I hear that "Vitalik is the CEO of Ethereum", Ethereum only have co-founders including Vitalik, Gav Wood, Joseph Lubin and several others. Groups or orgs like Parity, Consensys, Itering, DCDAO may have their own CXOs or managers cause they are structured according to their own constitution. These entities and individuals are hired by the network to serve or contributing with their interests.

Myself is contributing as chief architect and research for this project, more individually, also being voted as the council member of on-chain council pallet, I was also the co-founder of Itering, for other contributors, I think it's better for them to introduce by themselves or their organization.

**Reply by samestshares** [reply]
This is a great discussion and happy to see this ongoing action. I have been away for sometimes now but I am fully back. I am a Nigerian Ambassador and Competent Ambassador for Darwinia, wishing to contribute my own quota as I join the crew back. I am @Engr_Samest on TG

**Reply by SasoLithops** [reply]
Hello @hackfisher,

thank you for your time,warm welcome and detailed reply. I appologize will copy my tokenomics discussion to the thread you proposed I missed that one.

As for CXOs I always had/have community members in TG and in other chats(YoudleDAO, W3EI...) ask me they have no idea who is behind Darwinia. Thank you for detailed explanation. Will definitely copy this if questions come up. Make sense to me. It is weird for me also as it seems only Vitalik is ETH, but human mind is set to have hierarhy and ranks in companies. Hopefully our DAO will change that and about time we move to that mind set.

I am proud DCdao member. I will most certainly be active in community and discussions as I am invested in Darwinia since summer 2020. I have huge admiration for the tech Darwinia develops. As Megan once nicely said it speaks for itself.

**Reply by SasoLithops** [reply]
As for ideas I am also member on Ion DAO on Cosmos network where we try to build (it actually just launched) an index of cosmos ecosystem tokens.

https://index.ion.wtf/

It would be nice if we as DAO could sponsor a team to make a polkadot ecosystem tokens index and maybe get some index tokens in community pool as return.
Up to further disscussion if interest occurs. I can also connect you to the Ion team to maybe do a colaboration in future.

**Comment by SasoLithops** [comment]
Also to add I would suggest rewarding voters as having active governence is very important on proposals. I also suggest we add a quorum that must be reached for proposal to pass. For start we could have 5% as I think only few vote. Ammount of Kton could be used for voting and rewards could be in Kton as well(the one we get for staking Ring from community pool). Let's start giving Kton more utility. I haven't been active there yet too... 
And definitely give greater rewards to liquidity providers on DEXs we plan to list more then stakers as their risk is much greater. 
I would go as far as stakers get 10%(we already get rewarded when locking Ring,with receiving Kton) as they do not sevure the chain any more. Voters could get 5% Ring as bonus next to Kton. The rest 75% go to community pool which rewards liquidity providers with 15% value.  Do we have an address to our community pool so we can check the value? I would suggest we think of hackathlon soon so we unboard some defi projects or add additional support to Snowswap team which is doing great.

**Reply by ken1980-web** [reply]
Yes I agree kton have attraction because of low supply only but it would be great future if our developer can make some dapp for kton for usecase in term of adoption

**Comment by ken1980-web** [comment]
Many project restucted the tokonomics preparing next bullrun.

I researched some crypto ...I found 2 crypto changed their tokonomic which is better than before. I was much impressed. Example Cake and gala.. 
Yeah this is true that it is very  important to improve it according to current condition ☺️

I want to know if tokonomics could go inflation to deflation in term of maintaining the value?

**Comment by SasoLithops** [comment]
So I have done some more thinking. This is my opinion where I see the problem and the solution.

This was my post on TG when we discussed how much should go to stakers and how much to treasury.

"" I think if team provided some decent calculations on why they chose the ratio,and what would the actual drop be,as so many stakers that do nothing the numbers would be much better.Now it feels as these numbers are chosen randomly.
I would much rather have healthy and rich treasury and Ring priced 0,1$ then small treasury and Ring 0,001$.
Why receive 1000 Ring at 0,001$ and dropping when I can have 100 Ring at 0,1$ and rising.
There are so many variables in crypto,that our own interest should not blind us...it is only one variable....I look forward to team discussion on Github....""

I am sad to see it went exactlly like this. I am receiving twice the rewards(so 20% should actually be ok) and Ring price keeps tanking.

Decent math calculations based on many variables should be made. One variable math does not work here and calculations were proven wrong.
Our APY for staking is not sustainsble on the long run and it is another factor driving price down.

Let me explain.
I think our inflation is atm not too high only due to low prices(that goes for Crab especialy). Othervise it is too high.
I feel that we have newly minted Ring and on the other side poor usecases for Ring.

So Ring is minted from staking. What can we use it for:

- Staking
- Fees

What else? We have no deep liquidity pools on DEXs(hopefully incentivesed from both sides), it is not traded on many exchanges, no Defi with Ring, no NFT market....So I see the issue here. We need to make Ring more usable/functional. People are not willing to lock tokens for few years any more...crypto evolves too quickly...most don't want to lock it for Crowdloans and that is 2 years. They want massive returns in Crowdloan rewards to give it up for 2 years, which is hardly feasible nowadays....

Our APY for staking I also think is not sustainsble on the long run and it is another factor driving price down.

So I would consider looking into making us use Ring for other purposes where we can see it being taken out of market and put into different uses.

**Reply by SasoLithops** [reply]
You are right,but since English not my first language I am better at writting, because I have more time to think and correct many mistakes I make on the way😊

**Reply by Laki389** [reply]
> I am sad to see it went exactlly like this. I am receiving twice the rewards(so 20% should actually be ok) and Ring price keeps tanking.

These discussions here about Darwinia tokenomics are offtopic, but i will reply anyway.

- I HIGHLY doubt you have twice the staking rewards, unless you had a high Total Stake or high commission Validator in Darwinia 1.

- The staking rewards are slightly higher, but that's because not everyone migrated from Darwinia 1 to Darwinia 2, but you can't jump to conclusions about something that might change in the future. (there are still new Darwinia 2 migrations daily, which lowers your APY)

- The math is simple, the daily emission of a new amount of RING that goes to Collators and Delegates is reduced by 20%. Maybe you have a higher APY now, but the one who stakes with a collator with 107 668 858 Total Stake has a lower one, which means that the average is important, and the average is that the emission of a new RING (that goes to Collators and Delegates) is reduced by 20%.

**Reply by SasoLithops** [reply]
Why offtopic? I am talking about our inflation which corrected could make Darwinia great again.

To answer your first question:

I had 8.3M power and I received 1150Ring (validator had 10% commision, max one in set 20%).
Now migrating my power DROPPED with same ammount staked at 7,5M(collator 8-10%) and I receive just below 2000 Ring daily.
We are on blockchain so feel free to check.

As to second question. 

Answered above and what conclusions am I jumping too? I am discussing about why  many projects are changing inflation to survive. I in no way feel like all my opinions are right. But you are not proving your points to me with simple math. It doesn't work to me.Calculations were proven wrong.

As for third point it makes no sense to me why would that affects inflation. I understand average rewards...People sadly have to be very active in staking as our staking system is not stake and forget like Cosmos,Cardano....where validators rewards are stable, not like with us where there is constant power staked and commisions changes...I check every two days and was left without rewards twice and they dropped 4 times....But that is another problem in another thread...

I am sorry you don't understand I mean only well for Darwinia and us in the long run. 
Maybe you dream of that scenario:

Lets say more people stake. You get lets say 1500 Ring then. And price goes to 0.1$ So you receive 150$ daily rewards. Do you really think this would not again lead to huge price dump? And imagine what whales would get....there is no sustainibility in that scenario....I had that dream a lot, but staking can not offer that in any project... That is my opinion.

And at the end it would be great if you gave your ideas or solutions to make Darwinia great again. Or do you feel everything is ok and we leave it as it is?

**Reply by Laki389** [reply]
> Why offtopic? I am talking about our inflation which corrected could make Darwinia great again.

Becouse the topic of this discussion is https://github.com/dcdao/awesome-darwinia quote "This is a curated list of resources: utilities, examples, libraries and so on, for Darwinia ecosystem. It is for novice, user, and developer."
I understand https://github.com/dcdao/awesome-darwinia as a collection of useful links.

Also, @hackfisher emphasized with his comment on May 31, quote "This thread is for discussion about this repo maintenance, let's try to keep the discussions on topic. Off-topics should on separate threads.

For tokenomics, there is another RFC on darwinia-network/darwinia#1163, you can comment your ideas there." end of quote

**Reply by SasoLithops** [reply]
Oh ok. I forgot about that. Thanks Laki. Will copy it there too. I am so confused with this Github posts but that is completely on me🤦

**Reply by Laki389** [reply]
> To answer your first question:
> 
> I had 8.3M power and I received 1150Ring (validator had 10% commision, max one in set 20%). Now migrating my power DROPPED with same ammount staked at 7,5M(collator 8-10%) and I receive just below 2000 Ring daily. We are on blockchain so feel free to check.

- In Darwinia 1 nominators could choose 16 validators which resulted in the Total Stake not being as drastically different as in Darwinia 2 where only 1 collator could be chosen. In Darwinia 2, there are collators with 23M total stake, but there are also collators with 107M. Your example is based on I guess 27-30M, but to see the average let's choose a collator with 629M/15 = 41.93M and tell us the result, do you still have x2 (the question is rhetorical).

> As to second question.
> 
> Answered above and what conclusions am I jumping too? I am discussing about why many projects are changing inflation to survive. I in no way feel like all my opinions are right. But you are not proving your points to me with simple math. It doesn't work to me.Calculations were proven wrong.

- You are jumping to conclusions with your example where you chose 23-28M collator, choose from 42M and you will not get x2. Not everyone can choose the best collators.
What was proven wrong? In Darwinia 1 nominators and validators got 182 500 Ring/daily, now they all get 146 000. You get x2 more becouse someone gets x3 less or because some have not migrated yet.

> As for third point it makes no sense to me why would that affects inflation. I understand average rewards...People sadly have to be very active in staking as our staking system is not stake and forget like Cosmos,Cardano....where validators rewards are stable, not like with us where there is constant power staked and commisions changes...I check every two days and was left without rewards twice and they dropped 4 times....But that is another problem in another thread...

- I didn't say inflation has changed, inflation is the same, with now 36500 Ring per day less going to collators+delegates and 36500 Ring per day more going to the treasury.

> I am sorry you don't understand I mean only well for Darwinia and us in the long run. 

- The road to hell is paved with good intentions.

>Maybe you dream of that scenario:
> 
> Lets say more people stake. You get lets say 1500 Ring then. And price goes to 0.1$ So you receive 150$ daily rewards. Do you really think this would not again lead to huge price dump? And imagine what whales would get....there is no sustainibility in that scenario....I had that dream a lot, but staking can not offer that in any project... That is my opinion.
> 
> And at the end it would be great if you gave your ideas or solutions to make Darwinia great again. Or do you feel everything is ok and we leave it as it is?

- Annual inflation of 2%. Even better, every year a fixed amount of the current 2%, so every year a new 45 679 075 Ring. The first year it is 2%, the next year less than 2% and so on.
- Now, this is not enough and the only solution. Some project can be deflationary and the price of the coin can sink. In order for a project to be successful, it should have a product that is needed on the market and in demand. Does Darwinia have a product? I don't think there is yet.
- After the project has a product that is needed on the market, marketing is also needed. This means attending the most important gatherings like Polkadot Decoded, giving interviews for The Kusamarian (if there are funds and with significant crypto-global "influencers"), an aggressive presence on Twitter and so on.

**Reply by SasoLithops** [reply]
You are making it hypothetical(now you acting like politician). I was active in 1.0 and am active in 2.0 in staking. And I get double the rewards. And to make it even more real, 8 collators(half) have 10% or below commision and below 30M power.

This is turning into ping-pong debate and I see it as it has no end and is meaningles for improvement.

So I end it here. We agree to disagree🤝

Let's continue with your last three points in the correct thread now. Hope others join.

**Reply by Laki389** [reply]
> You are making it hypothetical(now you acting like politician). I was active in 1.0 and am active in 2.0 in staking. And I get double the rewards. And to make it even more real, 8 collators(half) have 10% or below commision and below 30M power.

Lol, I wouldn't call attaching numbers hypothetical. The fact is that 36 500 RING goes daily less to delegators and collators, which is 20% less. You want to build an opinion based only on your case, but let's ask those staking at 107M collator? Here, you choose that collator, you will start getting x4 times less staking rewards compared to the previous collator, or x2 times less compared to Darwinia 1, but the individual is not important, when polling agencies do polls they don't ask 1 person, they ask at least 1500 and form results. If a director in Slovenia receives a salary of 15,000 euros, that is not an average salary in Slovenia. But what are we even talking about here, the numbers are exact, 20% less goes to collators and delegates and that's it.

**Comment by SasoLithops** [comment]
Been doing some more researching regarding our inflation.

Astar gave great idea about dynamic inflation (below at the thread)

https://forum.astar.network/t/astar-tokenomics-2-0-a-dynamically-adjusted-inflation/4924

Astar has its dApps staking so different model, but maybe we could think in that direction too, to have it price orientated.

I think Cosmos Hub has it too,but its inflation is too high to take into consideration...

Let me know if we could go down that path,will do more research...

Imho Megan would be a great person to discuss this with too....

**Reply by SasoLithops** [reply]
It is a lot. I will need to dig deeper too.

You mean Astar consulted?

No I don't know any.
